feature-libs/quote/components/quote-list/quote-list.component.spec.ts

```diff
@@ -34,22 +34,26 @@ const mockPagination: PaginationModel = {
   pageSize: 5,
   sort: 'byCode',
 };
+
 const mockAction = { type: QuoteActionType.EDIT, isPrimary: true };
 const mockQuote: Quote = {
   ...createEmptyQuote(),
   allowedActions: [mockAction],
   cartId: mockCartId,
   code: '333333',
 };
+
 const mockQuoteList: QuoteList = {
   pagination: mockPagination,
   quotes: [mockQuote],
 };
+
 const mockQuoteListState: QueryState<QuoteList> = {
   loading: false,
   error: false,
   data: mockQuoteList,
 };
+
```

Comment:
is there a coding guideline that enforces us to enter an empty line after each const declaration?

Reply:
It's tiring and hard to find and understand something over time when the tests come in a row without a blank line.

Reply:
its not about tests, but about multiline const delcarations....
It's tiring and hard to find and understand something over time when everything is separated with a blank line.

Reply:
done